jest.unit.conf.json

```diff
@@ -3,7 +3,8 @@
     "\\.(css|less)$": "<rootDir>/frontend/test/__mocks__/styleMock.js",
     "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/frontend/test/__mocks__/fileMock.js",
     "^promise-loader\\?global\\!metabase-lib\\/lib\\/metadata\\/utils\\/ga-metadata$": "<rootDir>/frontend/src/metabase-lib/lib/metadata/utils/ga-metadata.js",
-    "ace/ext-searchbox": "<rootDir>/frontend/test/__mocks__/aceSearchBoxExtMock.js"
+    "ace/ext-searchbox": "<rootDir>/frontend/test/__mocks__/aceSearchBoxExtMock.js",
+    "^d3-(.*)$": "d3-$1/dist/d3-$1"
```

Comment:
Why do we need this settings for d3 libraries? Won't it compile without this setting?

Reply:
Exactly, tests won't compile